roles/deploy/tasks/main.yml

```diff
@@ -27,6 +27,11 @@
       repo: "{{ repo }}"
       dest: /home/{{ ansible_ssh_user }}/{{ project_name }}
 
+- name: install sqlite dependencies
```

Comment:
It seems like this should go in the packages, bundler, or a debian specific dependency role.